lib/infoblox.rb

```diff
@@ -13,7 +13,7 @@
 end
 
 module Infoblox
-  WAPI_VERSION   = '1.0'
+  WAPI_VERSION   = '1.1'
```

Comment:
Perhaps `ENV['WAPI_VERSION'] || '1.0'`?

In that case we would need a guard on objects that aren't in 1.0. Another way would be to configure it on a per-resource basis.